src/Controller/BlogController.php

```diff
@@ -57,6 +57,19 @@ public function tag(string $tag, int $page = 1, int $perPage = 20): Response
         ])->setLastModified(ContentUtils::max($pageArticles, 'lastModifiedOrCreated'));
     }
 
+    #[Route('/rss', name: 'blog_rss')]
```

Comment:
```suggestion
    #[Route('/rss.xml', name: 'blog_rss')]
```

?

Sinon il faudrait au moins ajouter une règle Nginx pour la prod pour le servir avec le bon content-type

Sur gh-pages actuellement : https://elao.github.io/elao_/pr/401/blog/rss